inputs/manual_input.csv

```diff
@@ -174,6 +174,20 @@ Ammonia cracker,investment,2050,527592.22,EUR/MW_H2,2015,"Ishimoto et al. (2020)
+methanol-to-olefins/aromatics,electricity-input,2015,1.3889,MWh_el/t_HVC,-,"DECHEMA 2017: DECHEMA: Low carbon energy and feedstock for the European chemical industry (https://dechema.de/dechema_media/Downloads/Positionspapiere/Technology_study_Low_carbon_energy_and_feedstock_for_the_European_chemical_industry.pdf), page 69","5 GJ/t_HVC" 
+methanol-to-olefins/aromatics,methanol-input,2015,18.03,MWh_MeOH/t_HVC,-,"DECHEMA 2017: DECHEMA: Low carbon energy and feedstock for the European chemical industry (https://dechema.de/dechema_media/Downloads/Positionspapiere/Technology_study_Low_carbon_energy_and_feedstock_for_the_European_chemical_industry.pdf), Sections 4.5 (for ethylene and propylene) and 4.6 (for BTX)","Weighted average: 2.83 t_MeOH/t_ethylene+propylene for 21.7 Mt of ethylene and 17 Mt of propylene, 4.2 t_MeOH/t_BTX for 15.7 Mt of BTX. Assuming 5.54 MWh_MeOH/t_MeOH." 
+methanol-to-olefins/aromatics,carbondioxide-output,2015,0.6107,t_CO2/t_HVC,-,"DECHEMA 2017: DECHEMA: Low carbon energy and feedstock for the European chemical industry (https://dechema.de/dechema_media/Downloads/Positionspapiere/Technology_study_Low_carbon_energy_and_feedstock_for_the_European_chemical_industry.pdf), Sections 4.5 (for ethylene and propylene) and 4.6 (for BTX)","Weighted average: 0.4 t_MeOH/t_ethylene+propylene for 21.7 Mt of ethylene and 17 Mt of propylene, 1.13 t_CO2/t_BTX for 15.7 Mt of BTX. The report also refere" 
+steam cracker,investment,2015,10512000,EUR/(t_HVC/h),2015,"DECHEMA 2017: DECHEMA: Low carbon energy and feedstock for the European chemical industry (https://dechema.de/dechema_media/Downloads/Positionspapiere/Technology_study_Low_carbon_energy_and_feedstock_for_the_European_chemical_industry.pdf), Table 35","Assuming CAPEX of 1200 €/t actually given in €/(t/a)."
+steam cracker,lifetime,2015,30,years,-,"Guesstimate",""
+steam cracker,FOM,2015,3,%/year,-,"Guesstimate",""
+steam cracker,VOM,2015,180,€/t_HVC,2015,"DECHEMA 2017: DECHEMA: Low carbon energy and feedstock for the European chemical industry (https://dechema.de/dechema_media/Downloads/Positionspapiere/Technology_study_Low_carbon_energy_and_feedstock_for_the_European_chemical_industry.pdf), Table 35",""
+steam cracker,naphtha-input,2015,14.8,MWh_naphtha/t_HVC,-,"Lechtenböhmer et al. (2016): 10.1016/j.energy.2016.07.110, Section 4.3, page 6.",""
+steam cracker,electricity-input,2015,2.7,MWh_el/t_HVC,-,"Lechtenböhmer et al. (2016): 10.1016/j.energy.2016.07.110, Section 4.3, page 6.","Assuming electrified processing."
+steam cracker,carbondioxide-output,2015,0.55,t_CO2/t_HVC,-,"DECHEMA 2017: DECHEMA: Low carbon energy and feedstock for the European chemical industry (https://dechema.de/dechema_media/Downloads/Positionspapiere/Technology_study_Low_carbon_energy_and_feedstock_for_the_European_chemical_industry.pdf), ","The report also references another source with 0.76 t_CO2/t_HVC"
```

Comment:
Are there other types of steam crackers (non-electric driven) and do they share the same CAPEX structure? If no we might want to call it something like `electric steam cracker`.

Reply:
Agree, it should be electric steam crackers.

https://www.basf.com/global/en/who-we-are/sustainability/whats-new/sustainability-news/2022/basf-sabic-and-linde-start-construction-of-the-worlds-first-demonstration-plant-for-large-scale-electrically-heated-steam-cracker-furnaces.html

Today, most is non-electric.